src/version/sagas/feedsRssSagas.js

```diff
@@ -0,0 +1,24 @@
+import { takeEvery, put } from 'redux-saga/effects';
+
+import { FEEDS_RSS_ACTION, feedsRssAction } from "../actions/feedsRssAction";
+import { getRssFeedsListFromXML } from "../plugins/Feeds/rss-helper";
+
+const CORS_PROXY = "https://cors-anywhere.herokuapp.com/";
```

Comment:
again, use window.config instead of constants like that

Reply:
Moved to rss URL